show/main.py

```diff
@@ -661,6 +680,32 @@ def portchannel(verbose):
     cmd = "sudo teamshow"
     run_command(cmd, display_cmd=verbose)
 
+#
+# 'subinterfaces' group ("show subinterfaces ...")
+#
+
+@cli.group(cls=AliasedGroup, default_if_no_args=False)
+def subinterfaces():
+    """Show details of the sub port interfaces"""
+    pass
+
+# 'subinterfaces' subcommand ("show subinterfaces status")
+@subinterfaces.command()
+@click.argument('subinterfacename', type=str, required=False)
+@click.option('--verbose', is_flag=True, help="Enable verbose output")
+def status(subinterfacename, verbose):
+    """Show sub port interface status information"""
+    cmd = "intfutil status"
+
+    if subinterfacename is not None:
+        if get_interface_mode() == "alias":
+            subinterfacename = iface_alias_converter.alias_to_name(subinterfacename)
+
+        cmd += " {}".format(subinterfacename)
```

Comment:
>subinterfacename [](start = 28, length = 16)

You are concat command line from user input, then always validate input to prevent attack.